dev/CommandBarFlyout/InteractionTests/CommandBarFlyoutTests.cs

```diff
@@ -4,6 +4,7 @@
 using Common;
 using System;
 using System.Collections.Generic;
+using System.Drawing;
```

Comment:
What's this used for?

Reply:
It's used for:
. Rectangle initialBoundingRectangle
. Rectangle finalBoundingRectangle